app/views/assignments/edit/_review_strategy.html.erb

```diff
@@ -24,6 +24,12 @@
     <br/>
       <%= label_tag('assignment_form[assignment][max_reviews_per_submission]', 'Maximum number of reviews per submission:') %>
       <%= text_field_tag('assignment_form[assignment][max_reviews_per_submission]', @assignment_form.assignment.max_reviews_per_submission ||= 10, size: 1) %>
+    <br/>
```

Comment:
This should be toggled, instructors should be able to give a) max # of reviews for each artifact or b) # reviews required for each student, but not both.  If we allow both to be displayed to instructors, it will cause more confusion.

Even better, one of those can be calculated by another as below:
max_#_of_reviews for_each_artifact * #_team/#_participant = #_reviews_required_for_each_student

If instructor fill in one of them, either a) or b), it would be better if you guys use javascript to remind instructor what is the other value suppose all the students do their work.